src/docs/asciidoc/wan.adoc

```diff
@@ -599,6 +599,10 @@ You can also use the following URL in your REST call if you want to synchronize
 `http://member_ip:port/hazelcast/rest/mancenter/wan/sync/allmaps`
 ====
 
+NOTE: Synchronizing WAN Target Cluster operates only with the data resides in the memory. Therefore, evicted entries will not be
```

Comment:
NOTE: Synchronization for a target cluster operates only with the data residing in the memory.